lib/ContentHelpers.php

```diff
@@ -25,8 +25,8 @@ public static function getCommonFields(Entry $entry, $status, $locale)
             'linkUrl' => $entry->externalUrl ? $entry->externalUrl : EntryHelpers::uriForLocale($entry->uri, $locale),
             'title' => $entry->title,
             'trailText' => $entry->trailText ?? null,
-            'hero' => $entry->heroImage ? Images::extractHeroImage($entry->heroImage) : null,
-            'heroCredit' => $entry->heroImageCredit ?? null,
+            // @TODO: Remove heroNew once main app is updated
+            'hero' => Images::buildHero($entry->hero),
```

Comment:
This might fix this issue, too: https://github.com/biglotteryfund/blf-alpha/issues/1748

Reply:
This change won't, but this should: https://github.com/biglotteryfund/blf-alpha/pull/1722/files#diff-981f3f59b5bbe0747c50a0bf4457da5eR62